### DISCUSSION TITLE
Customise appearance?

### DISCUSSION THREAD

**Discussion by crispybegs** [TriliumNext/Trilium#4131] [General]
Hi all, just started using trilium and really love it so far.

I was wondering if it's possible to customise the appearance beyond the light / dark theme? for example, having a totally different background colour / changing the logo top-left and so on.

Is that possible?

**Comment by zadam** [comment]
If you know CSS then it's possible and relatively easy to customize the look: https://github.com/zadam/trilium/wiki/Themes#custom-css

**Comment by nema69** [comment]
There are also a bunch of themes people made. Check out the theme gallery in the wiki.
